include/zephyr/kernel_structs.h

```diff
@@ -258,8 +258,11 @@ extern atomic_t _cpus_active;
  */
 bool z_smp_cpu_mobile(void);
 
-#define _current_cpu ({ __ASSERT_NO_MSG(!z_smp_cpu_mobile()); \
-			arch_curr_cpu(); })
+#define _current_cpu                                                                               \
```

Comment:
The current formatting looks better in the editor:

![Screenshot 2024-11-07 at 12 32 24 PM](https://github.com/user-attachments/assets/fe3a357b-d75a-4d86-bb72-da62b8cf104e)